src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page-content">
+    <section class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="image"
+        class="product-card__image"
+      >
+
+      <div class="product-info">
+        <div class="product-info__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+
+        <p class="product-info__code">
+          Код товара: 195434
+        </p>
+      </div>
+
+      <div class="rating">
+        <div class="rating__stars stars stars--active">
```

Comment:
Modifier must clearly identify the setting it modifies.
```suggestion
        <div class="rating__stars stars stars--4">
```